Doc/library/os.rst

```diff
@@ -2397,13 +2399,14 @@ features:
 
 .. function:: mknod(path, mode=0o600, device=0, *, dir_fd=None)
 
-   Create a filesystem node (file, device special file or named pipe) named
-   *path*. *mode* specifies both the permissions to use and the type of node
-   to be created, being combined (bitwise OR) with one of ``stat.S_IFREG``,
-   ``stat.S_IFCHR``, ``stat.S_IFBLK``, and ``stat.S_IFIFO`` (those constants are
-   available in :mod:`stat`).  For ``stat.S_IFCHR`` and ``stat.S_IFBLK``,
-   *device* defines the newly created device special file (probably using
-   :func:`os.makedev`), otherwise it is ignored.
+   Create a filesystem node (file, device special file, named pipe or socket)
+   named *path*. *mode* specifies both the permissions to use and the type of
+   node to be created, being combined (bitwise OR) with one of
+   ``stat.S_IFREG``, ``stat.S_IFCHR``, ``stat.S_IFBLK``, ``stat.S_IFIFO`` and
+   ``stat.S_IFSOCK`` (those constants are available in :mod:`stat`).  For
```

Comment:
If we document support for ``S_IFSOCK``, we should have a test for it.

Reply:
Since this MR only deals with the `sendfile()` orderings stuff now, that part is in principle obsolete.

But for the other issue (#107979), what kind of test would you have wanted to see? Something in `test_posix.py` like `test_mknod`?

OTOH, one could argue, that, since POSIX says all but `S_IFIFO` are non-portable, a test is anyway difficult to make. Plus there are already none for the other file types that are already documented.